packages/ethereum-contracts/contracts/upgradability/SuperfluidUpgradeableBeacon.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: AGPLv3
+pragma solidity 0.8.19;
+
+import {
+    UpgradeableBeacon
+} from "@openzeppelin/contracts/proxy/beacon/UpgradeableBeacon.sol";
+import { BeaconProxiable } from "./BeaconProxiable.sol";
+
+contract SuperfluidUpgradeableBeacon is UpgradeableBeacon {
```

Comment:
Note for the next review:

Consider to merge SuperfluidUpgradeableBeacon & BeaconProxiable into BeaconProxiable, similar to how UUPSProxy is so that more code can be in the upgradable area.